### ISSUE TITLE
setRate does not work when the duration of audio is less than 2 seconds on iOS

### ISSUE THREAD

**Issue by genglei01** [react-native-kit/react-native-track-player#966]
**Describe the bug**
setRate does not work when the duration of audio is less than 2 seconds on iOS

**To Reproduce**
Play a audio which duration is less than 2s.

**Environment (please complete the following information):**
"@ant-design/react-native": "^3.3.0",
"react-native-track-player": "^1.2.3",

**Comment by curiousdustin** [comment]
Which platform are you experiencing this issue on?

What is the actual behavior you see? Does it play at normal rate instead?

**Comment by genglei01** [comment]
@curiousdustin , yes, play at the normal rate(rate:1.0）and it works well(setRate(0.5)) on android.

**Comment by curiousdustin** [comment]
Real device or simulator?

Hosted file, local file path, or required/imported file?

File type? Are you able to share the file?

Are you able to share your code?

I just tested with a 2 second file and it works on simulator with local files paths, required files, and hosted files.

**Comment by genglei01** [comment]
@curiousdustin  thank you quick response.
I upload a demo  https://github.com/genglei01/ReactNativeTest.git
Refer  it, please

**Comment by curiousdustin** [comment]
I downloaded your `can_you_see.mp3` file, and tested it in my project. It works fine for me on simulator.

There must be something different in your code that is causing the issue.

A couple things you could try:

Try swapping these lines. Set the rate first.
```js
        await TrackPlayer.play();
        await TrackPlayer.setRate(this._speed);
```

Try adding await to these lines.
```js
    player.addTrace(audio1);
    player.addTrace(audio2);
    player.play()
```

**Comment by genglei01** [comment]
Yes, If only play one trace work fine. 
But  I want to add two traces and play them in my demo, and I update my code as your suggestion, only the first trace in the first time can play as  I set rate(0.5), the second trace play as speed 1. All traces work as speed 1 when tapping the `Play audio` button.(but the first trace works  sometimes when tapping the button)
Please run my demo and review it(I run it on simulator iPhone 11).
Thanks

**Comment by curiousdustin** [comment]
Oh. Now that you mention the issue only happens when playing multiple tracks it sounds like it is this bug: https://github.com/react-native-kit/react-native-track-player/issues/614

This has been fixed on the dev branch but has not been release yet. 

Also it has nothing to do with track duration.

**Comment by genglei01** [comment]
Yes, the issue is resolved when I update code as your commit f57f184 on Apr 30,
Is there any release schedule?
